spid_cie_oidc/onboarding/migrations/0003_onboardingregistrationform.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.0.2 on 2022-02-18 17:21
```

Comment:
this migration is quite useless

if you have changed your models.py please run again `makemigrations` and `migrate`
but please, remove this 0003 because it's useless